lib/cli-engine/cli-engine.js

```diff
@@ -934,9 +937,10 @@ class CLIEngine {
     /**
      * Checks if a given path is ignored by ESLint.
      * @param {string} filePath The path of the file to check.
+     * @param {boolean} isDot is the file a dot file.
      * @returns {boolean} Whether or not the given path is ignored.
      */
-    isPathIgnored(filePath) {
+    isPathIgnored(filePath, isDot) {
```

Comment:
This is part of our published API so it would be preferable not to change the function signature.